flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/FailureHandlingResult.java

```diff
@@ -204,13 +226,15 @@ public static FailureHandlingResult restartable(
             @Nullable Execution failedExecution,
             @Nullable Throwable cause,
             long timestamp,
+            CompletableFuture<Map<String, String>> labels,
```

Comment:
nit: there is an inconsistent naming of this parameter across this file

Reply:
Ack -- renamed all to the more descriptive `failureLabels`